OpenRA.Mods.Common/Traits/World/CreateMPPlayers.cs

```diff
@@ -73,9 +74,36 @@ void ICreatePlayers.CreatePlayers(World w)
 			w.SetPlayers(worldPlayers, localPlayer);
 
 			foreach (var p in w.Players)
+			{
```

Comment:
Think we need to reset the LongBitSet<PlayerBitMask>, otherwise we will continue to allocate new bits each game if internal name changes.

Reply:
If we moved this logic to the `World` ctor (see my earlier comment) that would also be a natural place to reinit the LongBitSet.